rero_ils/modules/patrons/templates/rero_ils/patron_profile.html

```diff
@@ -34,21 +35,31 @@
     <nav>
       <ul class="nav nav-tabs" id="nav-tab" role="tablist">
         <li class="nav-item">
-          <a {% if tab == 'checkouts' %}class="nav-link active"{%else%}class="nav-link"{% endif %} href="#checkouts" data-toggle="tab" id="checkouts-tab" title="{{ _('Loans') }}"
-            role="tab" aria-controls="checkouts" aria-selected="true">
-            {{ _('Loans') }} ({{ checkouts|length }})
+          <a {% if tab == 'loans' %}class="nav-link active"{%else%}class="nav-link"{% endif %} href="#loans" data-toggle="tab" id="loans-tab" title="{{ _('Loans') }}"
+            role="tab" aria-controls="loans" aria-selected="true">
+            {{ _('Loans') }}
+            <span class="badge badge-light font-weight-normal">{{ loans|length }}</span>
```

Comment:
COSMETIC : spaces before and after the `|`. Same at line 48

Reply:
done